docs/UserGuide.md

```diff
@@ -34,44 +31,36 @@ than current GUI apps in the industry.
 
 ---
 
-## Quick start
+## **Quick start**
 
 1. Make sure that you have **Java 11 or above** installed on your computer.
    - [How do I check my version of Java?](#checking-java-version)
-   <br><br><br>
 2. Download the latest jar file (`docedex.jar`) from our [Github Releases](https://github.com/AY2223S2-CS2103T-F12-1/tp/releases/).
    ![DownloadDocedex](images/DownloadDocedex.png)
-   <br><br><br>
 3. Move the jar file to a folder where you want to store the data of the doctors and patients.
 - Create a new folder called (you can name it Docedex) where you would like to house the application (e.g., Desktop, Documents, etc.).
 - Move the downloaded jar file to this folder as shown below.
-  - For Mac Users:<br>
-  <img src="images/DownloadDocedexMac.png" alt="DownloadDocedexMac" width="70%" height="70%">
-  - For Windows Users:<br>
-  <img src="images/DocedexInFileWin.png" alt="DocedexInFileWin" width="70%" height="70%">
```

Comment:
I put the size restrictions on the images as I felt that they were too big. Do let me know if you had any considerations in removing them!

Reply:
To improve visibility of images and readability of embedded content (on print, the images were too small and the text within was very hard to read).